project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.10.0"
 
-  private val workbenchLibsHash = "a6ad7dc"
+  private val workbenchLibsHash = "b57a1ca"
```

Comment:
Needed for `jackson-module-scala` update, to be compatible with the new WSM client.